test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex routing url
+   Given Minishift has state "Running"
+     When status code of HTTP request to "Routing" is equal to "200"
```

Comment:
This really needs to be a parameter. How does one know which route to access? There might and will be more than one at some times.